include/crpropa/magneticField/turbulentField/PlaneWaveTurbulence.h

```diff
@@ -0,0 +1,110 @@
+#ifndef CRPROPA_PLANEWAVETURBULENCE_H
+#define CRPROPA_PLANEWAVETURBULENCE_H
+
+#include "crpropa/Grid.h"
+#include "crpropa/magneticField/turbulentField/TurbulentField.h"
+#include <vector>
+
+namespace crpropa {
+/**
+ * \addtogroup MagneticFields
+ * @{
+ */
+
+/**
+ @class PlaneWaveTurbulence
+ @brief Interpolation-free turbulent magnetic field based on the GJ99 and TD13
```

Comment:
gj99 and td13 - lowercase - to be consistent

Reply:
I've changed everything to uppercase to be consistent with the rest of the source code.